protocol-designer/src/pages/Designer/ProtocolSteps/Timeline/TerminalItemStep.tsx

```diff
@@ -70,7 +74,7 @@ export function TerminalItemStep(props: TerminalItemStepProps): JSX.Element {
       <StepContainer
         {...{
           stepId: `TerminalItem_${id}`,
-          iconName: 'arrow-right',
+          iconName: handleIconName(),
```

Comment:
does it not work to directly inline do this?

```suggestion
          iconName: title === 'Starting deck state' ? 'ot-start' : 'ot-end',
```